src/styles/settings/_defaults.css

```diff
@@ -0,0 +1,12 @@
+*,
+*::before,
+*::after  {
+  --font-size-default: 20px;
+  --font-size-big:     30px;
+  --font-size-small:   10px;
+  --font-montserrat: 'Montserrat', sans-serif;
+
+  --zero-space:          0px;
+  --default-space:       15px;
+  --thirty-px-space:     30px;
```

Comment:
👍

Reply:
=D conforme o projeto for avançando vou criando mais variáveis e dando mais conteúdo, ainda está meio abstrato 

Obrigado pelas revisões !